rvic/core/param_file.py

```diff
@@ -49,8 +49,12 @@ def finish_params(outlets, dom_data, config_dict, directories):
     # subset (shorten time base)
     if options['SUBSET_DAYS'] and \
             options['SUBSET_DAYS'] < routing['BASIN_FLOWDAYS']:
-        subset_length = (options['SUBSET_DAYS'] *
+        subset_length = int(options['SUBSET_DAYS'] *
                          SECSPERDAY / routing['OUTPUT_INTERVAL'])
+        if subset_length % 2 != 0:
+            raise ValueError('either (1) SUBSET_DAYS / OUTPUT_INTERVAL or ',
+                             '(2) outlet unit hydrograph length must be a ',
+                             'multiple of 2')
```

Comment:
One suggestion is to print subset_length in this error message, along with subset_days and output_interval to aid in debugging.

Reply:
Done